php/lang/security/base-convert-loses-precision.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+$command = $secret = $counter = $uid = $totalColors = $i = $row = $value = $numeric = $string = '1';
+$getRandomNumber = 'rand';
+
+// ok: base-convert
```

Comment:
@Sjord all you annotations are using the wrong rule id: they should use `base-convert-loses-precision`.
That's why the CI check is failing :)

Reply:
Thanks. I fixed it. I renamed the rule to give it a more specific name, but apparently did not replace all occurrences.